environment.yml

```diff
@@ -0,0 +1,82 @@
+name: gene-ontology
+channels:
+  - defaults
+dependencies:
+  - blas=1.0=mkl
+  - ca-certificates=2018.03.07=0
+  - certifi=2018.10.15=py36_0
+  - decorator=4.3.0=py36_0
+  - intel-openmp=2019.0=118
+  - libedit=3.1.20170329=h6b74fdf_2
+  - libffi=3.2.1=hd88cf55_4
+  - libgcc-ng=8.2.0=hdf63c60_1
+  - libgfortran-ng=7.3.0=hdf63c60_0
+  - libstdcxx-ng=8.2.0=hdf63c60_1
+  - mkl=2019.0=118
+  - mkl_fft=1.0.6=py36h7dd41cf_0
+  - mkl_random=1.0.1=py36h4414c95_1
+  - ncurses=6.1=hf484d3e_0
+  - networkx=2.2=py36_1
+  - numpy=1.15.3=py36h1d66e8a_0
```

Comment:
Remove build numbers for all packages, i.e. 

```
- conda-forge::numpy=1.15.3
```

I also usually specify the channel as done with `conda-forge::`.

The build numbers have the effect of making envs restricted to a single operating system (e.g. mac or linux).

It's also probably better to only specify the versions of packages that we explicitly import, such as `pandas` but not behind-the-scenes packages that apply specially to certain architectures such as `intel-openmp`.

See the example at https://github.com/greenelab/manubot-rootstock/blob/dcfe402544f90e0047a247cdd8d01a2bca08df54/build/environment.yml